source/accessibility-statement.html.md.erb

```diff
@@ -0,0 +1,96 @@
+---
+title: Accessibility
+weight: 999
+hide_in_navigation: true
+---
+
+# Accessibility
+
+## Accessibility statement for data.gov.uk documentation
+
+This accessibility statement applies to the data.gov.uk technical documentation at [https://guidance.data.gov.uk/#data-gov-uk-technical-documentation](https://guidance.data.gov.uk/#data-gov-uk-technical-documentation).
```

Comment:
This can be simplified to:

```suggestion
This accessibility statement applies to the data.gov.uk technical documentation at <https://guidance.data.gov.uk/#data-gov-uk-technical-documentation>.
```

The same can be said for many of the links in the `Non-compliance with the accessibility regulations` section.

Reply:
cheers @ChrisBAshton - it's been peer-reviewed by tech writers and we've used this convention across all other accessibility statements that are being written so in this instance, I'm happy with it but will take this to the project's retro as it's a good point for future